docs/website/sidebars.js

```diff
@@ -383,6 +383,21 @@ const sidebars = {
             'running-in-production/tracing',
           ],
         },
+        {
+          type: "category",
+          label: "Run in Snowflake",
+          link: {
+            type: "generated-index",
+            title: "Run in Snwoflake",
+            description: "How to run dlt in Snowflake.",
+            slug: "walkthroughs/run-in-snowflake",
+            keywords: ["snowflake"]
+          },
+          items: [
+            "walkthroughs/run-in-snowflake/run-in-snowflake",
+            "walkthroughs/run-in-snowflake/application-architecture"
```

Comment:
There is a lot of deeply technical information about the app. Does it say the docs should be public? Maybe we can hide it from the sidebar for now?